utils/create-database.js

```diff
@@ -0,0 +1,72 @@
+// utils/create-database.js
```

Comment:
All comments here could be removed, especially the top line with the name of the file! The general purpose of comments is to inform the reader what you are doing rather than how you are doing it, especially in the cases where the code is not able to explain itself succinctly.